vuepress/docs/next/docs/releases/README.md

```diff
@@ -74,14 +74,16 @@ The Entando Hub is a new ECR feature that allows different EntandoApps to share
 * Bundle improvements; in particular, bundle plugin security and reliability via name signing.
 * It's now possible to specify standard and secret [environment variables](../../tutorials/devops/plugin-environment-variables.md) for a bundle plugin (microservice).
 * Now offers experimental support for [bundle plugin configuration profiles](../../tutorials/devops/plugin-configuration.md).
-* Now offers better support for bundles and bundle plugins with very long names.
+* Now offers better support for bundles and bundle plugins with very long names. 
+* ENTANDO_PLUGIN_NAME has been renamed to ENTANDO_RESOURCE_NAME.
 * Implemented better management of certain error conditions during installation and updates.
 * Introduced system-level bundles, which allow the installation of global static resources.
 * The ECR client now explicitly marks bundles that are installed with an amended installation plan and allows the plan to be recalled.
 * Added support for Oracle 18c.
 * Fixed a bug that caused the ECR to indefinitely stay in busy-state due to particular failure conditions.
 * Fixed a bug that (under certain conditions) prevented the bundle plugin pod from properly restarting when scaled down, then back up again.
 * The ECR now correctly deletes the plugin pod when the bundle is uninstalled, but the deployment and volumes are retained.
+* The EntandoPlugin controller will rollback in the event of a timeout. A failing pod in an OpenShift 4.8 installation triggers error logging and the deployment scales to 0.
```

Comment:
I don't think this is specific to OpenShift 4.8. I thought this was a general feature now. @w-caffiero-entando can you confirm?

Reply:
@nshaw sorry for the confusion; i think this was pulled from the acceptances criteria in https://entando.myjetbrains.com/youtrack/issue/ENG-3272

Reply:
@nshaw ok to remove the openshift reference?